Common/UnitDefinitions/CoefficientOfThermalExpansion.json

```diff
@@ -84,9 +78,6 @@
     {
       "SingularName": "PpmPerDegreeFahrenheit",
       "PluralName": "PpmPerDegreeFahrenheit",
-      "BaseUnits": {
-        "Θ": "DegreeFahrenheit"
-      },
```

Comment:
Ok so it seems that was not reason for removing these. So I guess duplicates are fine, but it also feels a bit icky that we don't control what units you get when requesting an SI unit, we just pick "the first one". 🤔 

If we had a policy of no duplicates, then at least it would be consistent what unit you get, but maybe there are good reasons for having multiple units specify the same base units?

Reply:
Yes, I've been thinking of different ways we could use the `BaseUnits` to test both the conversion coefficients and the provided `BaseUnits` (using the definitions from the summary comment) - but if we did that now, we'd have to account for the possible rounding errors (without a proper sense of scale etc..) - I think we're Ok for now so I'd say we do this later..

Reply:
No, we have to remove it, because the `PartsPerMillion` coefficient which multiplies the `BaseUnit`

Reply:
Ok, let's revisit later then